src/main/java/me/serce/solidity/ide/settings/SoliditySettingsConfigurable.form

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form xmlns="http://www.intellij.com/uidesigner/form/" version="1" bind-to-class="me.serce.solidity.ide.settings.SolidityConfigurablePanel">
+  <grid id="27dc6" binding="myEvmPathPanel" layout-manager="GridLayoutManager" row-count="3" column-count="3" same-size-horizontally="false" same-size-vertically="false" hgap="-1" vgap="-1">
+    <margin top="0" left="0" bottom="0" right="0"/>
+    <constraints>
+      <xy x="20" y="389" width="615" height="91"/>
+    </constraints>
+    <properties/>
+    <border type="none"/>
+    <children>
+      <component id="81eb4" class="javax.swing.JLabel">
+        <constraints>
+          <grid row="0" column="0" row-span="1" col-span="2" vsize-policy="0" hsize-policy="0" anchor="8" fill="0" indent="0" use-parent-layout="false"/>
+        </constraints>
+        <properties>
+          <labelFor value="539ac"/>
+          <text value="&amp;Path to EthereumJ VM:"/>
```

Comment:
@mbolotov Could you specify what needs to be provided here? I tried to specify the path to the ethereumj jar, but it wasn't enough as it was failing with ClassNotFound.

Reply:
It's pretty easy:
1. clone https://github.com/ethereum/ethereumj
2. `gradle shadowJar`
3. select `ethereumj/ethereumj-core/build/libs/ethereumj-core-1.7.0-SNAPSHOT-all.jar` file

Reply:
@mbolotov That's what I'm worried about. I don't think that any users of the plugin will figure out what they need to do, and will make a fat jar to make this work. Even if they're provided with documentation, I doubt that anyone will try to build a fat jar for this.

Reply:
It doesn't stop this from being merged, so I'll try to test it, and merge after if everything is ok, but we need to find a way to make this user-friendly, or else it wouldn't make much sense to provide the feature.

Reply:
Do you think putting the fat jar somewhere to download will be acceptable for such a user?
On the other hand, we could simply incorporate fat jar into the plugin itself. But I'm not sure if this is the best solution

Reply:
BTW, I found the execution failed when using an import statement. We might have to compile all the sources on the plugin side

Reply:
> Do you think putting the fat jar somewhere to download will be acceptable for such a user?

I think IJ should be able to download jar/deps dynamically and attach it to the classpath.